queries/angular/highlights.scm

```diff
@@ -1,12 +1,33 @@
+; inherits: html_tags
+
 (identifier) @variable
-(pipe_sequence "|" @operator)
+(pipe_operator) @operator
 (string) @string
 (number) @number
-(pipe_call
-  name: (identifier) @function)
-(pipe_call
-  arguments: (pipe_arguments
-               (identifier) @parameter))
+ (pipe_call
+   name: (identifier) @function)
+ (pipe_call
+   arguments: (pipe_arguments
+                (identifier) @parameter))
+
+(structural_directive
+  ["*"] @keyword
+  (identifier) @keyword
+)
+
+(attribute
+  (attribute_name) @variable.template
+  (#lua-match? @variable.template "#.*"))
```

Comment:
Why not `@property` like it was before?

Reply:
Because this is a local variable in the template. Component properties are highlighted as `@property`, but I wanted to highlight template variables differently.